packages/react-storage/src/components/StorageBrowser/Views/Controls/Download.tsx

```diff
@@ -42,6 +43,9 @@ function download(fileName: string, url: string) {
 }
 
 export const DownloadControl: DownloadControl = ({ fileKey }) => {
+  const getConfig = useGetLocationConfig();
+  const { bucket: bucketName, credentialsProvider, region } = getConfig();
```

Comment:
nit: any reason to rename bucket?

Reply:
yes you're right, sorry about that. Was just a copy paste I did while trying to fix the tests